spec/models/miq_expression/model_details_spec.rb

```diff
@@ -82,6 +82,10 @@
         result = described_class.model_details("Service", :typ => "all", :include_model => false, :include_tags => true)
         expect(result.map(&:first)).to include("My Company Tags : Auto Approve - Max CPU")
       end
+
+      it "Supports classes derived form ActsAsArModel" do
+        described_class.model_details("Chargeback", :typ => "all", :include_model => false, :include_tags => true)
```

Comment:
@gtanzillo Can you change this test to actually verify the results from model_details some way?